addon/bootstrap.js

```diff
@@ -130,3 +164,68 @@ function handleMessage(msg, sender, sendReply) {
+    }
+  }`;
+  const cssEnc = encodeURIComponent(css);
+  const newURIParam = {
+    aURL: 'data:text/css,' + cssEnc,
+    aOriginCharset: null,
+    aBaseURI: null
+  };
+  // implicit global assignment.
+  buttonStyleSheet = Services.io.newURI(newURIParam.aURL, newURIParam.aOriginCharset, newURIParam.aBaseURI);
+  styleSheetService.loadAndRegisterSheet(buttonStyleSheet, styleSheetService.AUTHOR_SHEET);
+}
+
+// borrowed from PocketContextMenu in m-c
+const contextMenu = {
+  init: function() {
+    Services.obs.addObserver(this, "on-build-contextmenu");
+  },
+  shutdown: function() {
+    Services.obs.removeObserver(this, "on-build-contextmenu");
+    for (let win of CustomizableUI.windows) {
+      const item = win.document.getElementById("context-screenshots");
+      if (item) {
+        item.remove();
+      }
+    }
+  },
+  observe: function(aSubject, aTopic, aData) {
+    const document = aSubject.wrappedJSObject.menu.ownerDocument;
+    let item = document.getElementById("context-screenshots");
+    if (!item) {
+      item = document.createElement("menuitem");
+      item.setAttribute("id", "context-screenshots");
+      item.setAttribute("label", "screenshots"); // TODO solve l10n without webextensions :-(
+      item.setAttribute("oncommand", "window.alert('clicked screenshots context menu')");
+      // TODO: add an "accesskey" attribute?
```

Comment:
Would that be a hotkey like asked for in https://bugzilla.mozilla.org/show_bug.cgi?id=1365757 ? Or just a key specific to the context menu?